core/templates/dev/head/pages/exploration_editor/add_exploration_metadata_modal.html

```diff
@@ -0,0 +1,58 @@
+<script type="text/ng-template" id="modals/addExplorationMetadata">
```

Comment:
Has the existing code for this modal been deleted? Alternatively, you could get rid of this file and deal with it in a separate PR if it's related to @Keksike's changes.